content/applications/finance/payment_providers/xendit.rst

```diff
@@ -0,0 +1,45 @@
+======
+Xendit
+======
+
+`Xendit <https://www.xendit.co>`_ is an Indonesian-based payment solution provider that covers
+several Southeast Asian countries. It allows businesses to accept credit cards as well as several local
+payment methods.
+
+.. _payment_providers/xendit/configure_dashboard:
+
+Configuration on the Xendit Dashboard
+=====================================
+
+#. Create a Xendit account if necessary and log in to the `Xendit Dashboard
+   <https://dashboard.xendit.co>`_.
+#. Check your account mode in the top left corner of the page. Use the :guilabel:`Test Mode` to try the integration
+   without charging your customers. Switch to :guilabel:`Live Mode` once you are ready to accept
+   payments.
+#. Navigate to :menuselection:`Configuration: Settings` in the left part of the application page. In the :guilabel:`Developers` section,
+   click `API Keys <https://dashboard.xendit.co/settings/developers#api-keys>`_. In tis page, you will setup for both :guilabel:`API key`
+   and :guilabel:`webhook token`.
+#. Click :guilabel:`Generate Secret Key`, and in the popup box, enter any :guilabel:`API key name`, select :guilabel:`Write` for
+   the :guilabel:`Money-in Products` permission and :guilabel:`None` for all other permissions
+   to keep your data secured, then click :guilabel:`Generate key`.
+#. Confirm your password to display your API key. Copy or download the key and **save
+   this information securely for later**. This is the only time the API key can be viewed or downloaded.
+#. When completed, scroll down the page to the `Webhooks <https://dashboard.xendit.co/settings/developers#webhooks>`_ section.
```

Comment:
```suggestion
#. Once completed, scroll down the page to the `Webhooks <https://dashboard.xendit.co/settings/developers#webhooks>`_ section to generate the webhook token.
```
(added the last part since i removed it from the next bullet point and i thought it would work well here)